deeppavlov/metrics/squad_metrics.py

```diff
@@ -40,6 +40,30 @@ def exact_match(y_true: List[List[str]], y_predicted: List[str]) -> float:
     return 100 * EM_total / len(y_true) if len(y_true) > 0 else 0
 
 
+@register_metric('exact_match_1.1')
+def exact_match_v1(y_true: List[List[str]], y_predicted: List[str]) -> float:
```

Comment:
naming should describe differences between similar metrics

Reply:
fixed